driver-core/src/test/java/com/datastax/driver/core/CodecRegistryTest.java

```diff
@@ -0,0 +1,77 @@
+package com.datastax.driver.core;
+
+import java.util.List;
+
+import com.google.common.reflect.TypeToken;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import static com.datastax.driver.core.Assertions.assertThat;
+
+public class CodecRegistryTest {
+    TypeToken<List<Integer>> LIST_OF_INT_TOKEN = new TypeToken<List<Integer>>() {
+    };
```

Comment:
Intellij's autoformat adds the line break, I'm not sure if that can be configured otherwise.